### PR TITLE
Likes

### PR DESCRIPTION
**Pull request by Zoneam**

### CODE REVIEW

File: server/schemas/typeDefs.js [hunk 1]
```diff
@@ -1,47 +1,47 @@
...
+  type Post {
+    _id: ID
+    title: String
+    body: String
+    author: String
+    dateCreated: String
+    comments: [Comment]
+    likes: [String]
+    totalLikes: Int
+    label: Label
```
Comment (melinamboedecker) [inline server/schemas/typeDefs.js:36]: Missing line above 35 that should be:
category: Category

Comment (Zoneam) [reply server/schemas/typeDefs.js:36]: fixed line 35

**Review by melinamboedecker — APPROVED** [review]

File: server/models/index.js [hunk 2]
```diff
@@ -1,6 +1,4 @@
 const User = require('./User');
-const Label = require('./Label');
 const Post = require('./Post');
-const Category = require('./Category');
 
-module.exports = { User, Label, Post, Category };
+module.exports = { User, Post, };
```
Comment (vasylynash) [inline server/models/index.js]: Remove trailing comma

File: server/schemas/resolvers/likes.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
+const Post = require('../../models/Post');
+
+module.exports = {
+  Query: {
+    postLikes: async (_, args) => {
+       const post = await Post.findById(args.id);
+       post.totalLikes = post.likes.length
+      return await post
```
Comment (vasylynash) [inline server/schemas/resolvers/likes.js]: Are you sure await is needed here? Also, add semicolons to the code

File: server/schemas/resolvers/likes.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+    },
+  },
+
+  Mutation: {
+    addPostLike: async (_, args) => {
+      const post = await Post.findById(args.id);
+
+      if (!post.likes.includes(args.userId)){
+      post.likes.push(args.userId)
```
Comment (vasylynash) [inline server/schemas/resolvers/likes.js]: Semicolons

**Review by vasylynash — CHANGES_REQUESTED** [review]
I don't understand why we need a separate field for the number of likes if it's just the length of likes array?

File: server/schemas/resolvers/likes.js (hunk 3 above)
Comment (Zoneam) [reply server/schemas/resolvers/likes.js]: yes, it doesn't work without await

**Review by vasylynash — COMMENTED** [review]
Also, fix the seeds.js

**Review by vasylynash — APPROVED** [review]

**Pull request merged**